labonneboite/web/contact_form/views.py

```diff
@@ -103,6 +104,24 @@ def unknown_siret_message():
 def get_office_identification_data():
     return forms.OfficeHiddenIdentificationForm(data=request.args, meta={'csrf': False}).data
 
+def sendMail(mail_content, subject):
+    '''
+        This funciton will send an email to us and retry in case of failure
+        The wait time will be 15s at worst
+    '''
+    max_attempts = 3
+    for retry in range(max_attempts):
+        try:
+            mail.send_mail(mail_content=mail_content, subject=subject)
+        except MailjetAPIError as e:
+            logger.exception(e)
+            if(retry < max_attempts - 1):
+                time.sleep(5*(retry+1))
+        else:
+            break
+    else:
+        logger.critical('Mail not sent (to settings.TO_EMAIL)', subject, mail_content)
+        flash(generate_fail_flash_content(), 'error')
```

Comment:
TIL for loops also have an else clause which most of us are unfamiliar with. The else clause executes after the loop completes normally. This means that the loop did not encounter a break statement. 

Plein de joli code pythonique ici, well played :-)